test/unit/account/search_test.rb

```diff
@@ -133,8 +133,10 @@ class Account::SearchTest < ActiveSupport::TestCase
 
   # For mysterious reason, the thing didn't use to work when indifferent hash was passed in.
   test 'search without query works with options with indifferent access' do
-    assert_nothing_raised do
-      results = Account.search(nil, ActiveSupport::HashWithIndifferentAccess.new)
+    ThinkingSphinx::Test.rt_run do
+      assert_nothing_raised do
```

Comment:
`assert_nothing_raised` is not needed, I think rubocop complained about it that it is unnecessary, just another tiny nit :angel:

Reply:
On the other hand - this PR hasn't added this `assert_nothing_raised`... And we use it in multiple tests.

I'd vote for keeping it as it is, and if we really want - refactor all tests at once to remove `assert_nothing_raised`. Again, if we really think we should do it. I personally think it's good to have it, as it helps understanding the test :shrug: 

I don't see a problem in having less debugging info for an already passing test. If it starts failing - it's very easy to remove the method and have extended debuggin info to figure out what's wrong.

Reply:
It was a suggestion. It's fine with me if you like to keep it.

Reply:
I'm for keeping it